.husky/pre-commit

```diff
@@ -2,3 +2,4 @@
 . "$(dirname -- "$0")/_/husky.sh"
 
 yarn exec lint-staged
+npm run lint
```

Comment:
use yarn here

Reply:
why lint-staged *and* run lint?

Reply:
Yeah `yarn exec lint-staged` should be enough